### ISSUE TITLE
Value for dynamo_backend not passed to `accelerate launch` after running `accelerate config`

### ISSUE THREAD

**Issue by Harvester62** [d8ahazard/sd_dreambooth_extension#1100]
## ⚠️If you do not follow the template, your issue may be closed without a response ⚠️
Kindly read and fill this form in its entirety.

### 0. Initial troubleshooting
Please check each of these before opening an issue. If you've checked them, **delete this section of your bug report**. Have you:
- Updated the Stable-Diffusion-WebUI to the latest version?
- Updated Dreambooth to the latest revision?
- Completely restarted the stable-diffusion-webUI, not just reloaded the UI?
- Read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?

### 1. Please find the following lines in the console and paste them below.

```
#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Dreambooth revision: 78ca4dabe9b269b7dae25960121c58c41e6c39e1
SD-WebUI revision: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2

[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.
[+] xformers version 0.0.17.dev464 installed.
[+] accelerate version 0.17.1 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.27.2 installed.
[+] bitsandbytes version 0.35.4 installed.

#######################################################################################################
```

### 2. Describe the bug

I don't know if it's a bug, no crash occurred but I didn't trained anything yet. A warning about dynamo_backend is displayed in the console at startup of Web UI despite having run 'accelerate config' twice, before starting the Web UI, one time in a "normal" console inside the \Scripts folder and a second time after activating the venv, following the entire procedure and providing all the required inputs.
Thank you.

**Screenshots/Config**
This is the content of the webui-user.bat file:

@echo off
REM git pull
pause

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS=--xformers --opt-split-attention --disable-safe-unpickle --api --allow-code --administrator --medvram --theme dark  
REM --cors-allow-origins=http://localhost:5173

SET XFORMERS_PACKAGE="xformers==0.0.17.dev447"
SET "REQS_FILE=.\extensions\sd_dreambooth_extension\requirements.txt"
SET DREAMBOOTH_SKIP_INSTALL=True
SET ACCELERATE="True"

call webui.bat

### 3. Provide logs

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

```st
venv "C:\stable-diffusion-webui\venv\Scripts\Python.exe"
Checking for accelerate
Accelerating
[20:23:51] WARNING  The following values were not passed to `accelerate launch` and had defaults used      launch.py:887
                    instead:
                            `--dynamo_backend` was set to a value of `'no'`
                    To avoid this warning pass in values for each of the problematic parameters or run
                    `accelerate config`.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2
```

### 4. Environment

What OS?
Windows 10 Pro 64 bit

If Windows - WSL or native?
Native

What GPU are you using?
NVIDIA GTX 1070 (8GB VRAM)

**Comment by ArrowM** [comment]
I'll take a look at this later (dynamo doesnt work on windows anyways tho, [source](https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/eval_frame.py#L411))

**Comment by ArrowM** [comment]
Alright, I looked a little. If you want a quick fix, you can probably manually change `webui.bat:59`. The code that handles accelerate launching is part of the A1111 project and not part of this dreambooth extension, so please open an issue in the A1111 repo for further support.

**Comment by Harvester62** [comment]
Thank you for looking into this. I will look into webui.bat at line 59 and see what I can change.
